### PR TITLE
Header & Community Update

### PR DESCRIPTION
**Pull request by phoban01**
## Description

This PR adds a Slack logo to the navigation bar and updates how we handle logos to make it more straightforward to add further items to the social menu in future.

We also have updated the community page with details about github, slack and community meetings.

Part of #122 
Part of #141 

Signed-off-by: Piaras Hoban <phoban01@gmail.com>

### CODE REVIEW

File: content/en/project/community.md [hunk 1]
```diff
@@ -11,4 +11,26 @@ menu:
...
 ---
 
-Please consult our [community repository](https://github.com/open-component-model-community) for details about the Open Component Model Community.
+## GitHub
+
+You can stay updated with OCM's latest advancements, join our active conversations, and delve into our enhancement proposals on each project's GitHub issues page specifically for OCM.
+
+If you're just starting with OCM, we recommend beginning with the 'good first issue' label in our repositories. 
+
+Ready to contribute directly to OCM? Whether it's adding code, writing tests, or assisting with our documentation, please adhere to the guidelines provided in the 'contributing' documentation of the relevant OCM repository.
```
Comment (Skarlso) [inline content/en/project/community.md:20]:
```suggestion
Ready to contribute directly to OCM? Whether adding code, writing tests, or assisting with our documentation, please adhere to the guidelines provided in the 'contributing' documentation of the relevant OCM repository.
```

File: content/en/project/community.md [hunk 2]
```diff
@@ -11,4 +11,26 @@ menu:
...
+
+## Slack
+
+Join #open-component-model on Kubernetes Slack and talk to us and over 1,000 other community members.
```
Comment (Skarlso) [inline content/en/project/community.md:24]:
```suggestion
Join [#open-component-model](https://kubernetes.slack.com/archives/C05UWBE8R1D) on Kubernetes Slack and talk to us and over 1,000 other community members.
```

**Review by Skarlso — APPROVED** [review]
LGTM with two suggestions.

File: content/en/project/community.md [hunk 3]
```diff
@@ -11,4 +11,26 @@ menu:
...
+
+You can stay updated with OCM's latest advancements, join our active conversations, and delve into our enhancement proposals on each project's GitHub issues page specifically for OCM.
+
+If you're just starting with OCM, we recommend beginning with the 'good first issue' label in our repositories. 
+
+Ready to contribute directly to OCM? Whether it's adding code, writing tests, or assisting with our documentation, please adhere to the guidelines provided in the 'contributing' documentation of the relevant OCM repository.
+
+## Slack
+
+Join [#open-component-model](https://kubernetes.slack.com/archives/C05UWBE8R1D) on Kubernetes Slack and talk to us and over 1,000 other community members.
```
Comment (phoban01) [inline content/en/project/community.md:24]:
```suggestion
Join [#open-component-model](https://kubernetes.slack.com/archives/C05UWBE8R1D) on Kubernetes Slack and talk to us and other community members.
```

**Pull request merged**